packages/components/accordion/src/Accordion.styles.ts

```diff
@@ -0,0 +1,19 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import type { CSSObject } from '@emotion/serialize';
```

Comment:
This dependency is not specified in package.json of the package, so it's better not to use this type